libs/server-sdk/include/launchdarkly/server_side/config/builders/data_system/data_system_builder.hpp

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <launchdarkly/server_side/config/builders/data_system/background_sync_builder.hpp>
+#include <launchdarkly/server_side/config/builders/data_system/lazy_load_builder.hpp>
+#include <launchdarkly/server_side/config/built/data_system/data_system_config.hpp>
+
+#include <launchdarkly/error.hpp>
+
+namespace launchdarkly::server_side::config::builders {
+
+class DataSystemBuilder {
+   public:
+    DataSystemBuilder();
+    using BackgroundSync = BackgroundSyncBuilder;
+    using LazyLoad = LazyLoadBuilder;
+
+    DataSystemBuilder& Disabled(bool disabled);
```

Comment:
Right now the status quo is to have an `offline` boolean that disables the data source + events. I was thinking it'd be nice to have a clearly labeled `Disabled()` call for just the sources:

```c++
auto cfg = ConfigBuilder("sdk123")
                      .DataSystem().Disabled(true)
```

That way we can say "Offline mode is shorthand for calling Events().Disabled() and DataSystem.Disabled()" or similar.